### DISCUSSION TITLE
Indonesian Translation not availabe?

### DISCUSSION THREAD

**Discussion by aspectxlol** [ChurchApps/ChurchAppsSupport#390] [FreeShow] [answered]
Hello!, i am having trouble trying to find a Indonesian "Terjemahan Baru" version of the bible in any format
[bible.com](https://www.bible.com/bible/306/GEN.1.TB?fbclid=IwZXh0bgNhZW0CMTAAAR2Ey1CUmJPr-o6OGI8S96_afqr3h_1wqaF-ThZmTmQ3y5swQ7E4DZnyZNU_aem_NpSrecl4v9oSeUeDGYl6WA)
this version is the version we mostly use at my church
can someone please help?
Thanks!

edit:
I did find this [repo](https://github.com/msiahaan/sword_modules) i dont know if its usable tho

**Comment by vassbo** [comment, marked as answer]
You should be able to find some here: https://freeshow.app/resources#scriptures
